camus-etl-kafka/src/main/java/com/linkedin/camus/etl/kafka/common/EtlCounts.java

```diff
@@ -45,7 +46,7 @@
   private long endTime;
   private long lastTimestamp;
   private long firstTimestamp = Long.MAX_VALUE;
-  private HashMap<String, Source> counts;
+  protected HashMap<String, Source> counts;
```

Comment:
changed to protected because it needs to be accessed in subclass EtlCountsForUnitTest in unit test.